packages/babel/src/index.js

```diff
@@ -31,11 +31,19 @@ const unpackOptions = ({
 
 const unpackInputPluginOptions = ({ skipPreflightCheck = false, ...rest }, rollupVersion) => {
   if (!rest.babelHelpers) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      "babelHelpers: 'bundled' option was used by default. It is recommended to configure this option explicitly, read more here: " +
-        'https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers'
-    );
+    if ('runtimeHelpers' in rest) {
```

Comment:
IMHO this could be moved to the top of this function - one probably won't have both `babelHelpers` & `runtimeHelpers` set at the same time, so it doesn't matter much but less indentation the better.